Algorithm.CSharp/BaseBitcoin.cs

```diff
@@ -0,0 +1,87 @@
+﻿using NodaTime;
+using QuantConnect.Brokerages;
+using QuantConnect.Data.Market;
+using QuantConnect.Securities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuantConnect.Algorithm.CSharp
+{
+
+    /// <summary>
+    /// Base class for Bitcoin algorithms. This base class provides standard behaviour for trading BTCUSD on Bitfinex
+    /// trading on Bitfinex
+    /// </summary>
+    public abstract class BaseBitcoin : QCAlgorithm
```

Comment:
I've provided the sample algorithm to show the required initialization for the brokerage. I'd appreciate advice on a cleaner method to do this. Perhaps a custom BrokerageSetupHandler?